test/integration.test.ts

```diff
@@ -1,12 +1,16 @@
-import { contains, lt } from '../client/src';
-import { XataClient } from '../codegen/example/xata';
+import { contains, lt, Repository } from '../client/src';
+import { User, XataClient } from '../codegen/example/xata';
+import { mockUsers } from './mock_data';
 
 const client = new XataClient({
   databaseURL: process.env.XATA_DATABASE_URL || '',
   branch: process.env.XATA_DATABASE_BRANCH || '',
   apiKey: process.env.XATA_API_KEY || ''
 });
 
+// TODO: Implement a bulk delete for teardown
+jest.setTimeout(50000);
+
```

Comment:
Integration tests will be time consuming, not sure if we want to increase the default ``timeout`` for jest or it's fine to only have it here.

Reply:
No strong opinions here. But we should update the comment if this won't be fixed with bulk deletes :)